Gemfile

```diff
@@ -6,7 +6,7 @@ gemspec
 
 gem 'appraisal'
 gem 'bump',          '~> 0.6'
-gem 'bundler',       '~> 1.17'
+gem 'bundler',       '~> 2.2'
```

Comment:
We can remove this line from the gemfile. There no need to lock the Gemfile to a specific bundler version anymore since bundler 1.17

Reply:
Fixed in 4fac6d973ba71ee5c40b5c7371c0dc5355818761